test/forms/WhatIsGIINFormProviderSpec.scala

```diff
@@ -21,34 +21,51 @@ import play.api.data.FormError
 
 class WhatIsGIINFormProviderSpec extends StringFieldBehaviours {
 
-  val requiredKey = "whatIsGIIN.error.required"
-  val lengthKey   = "whatIsGIIN.error.length"
-  val maxLength   = 100
-
   val form = new WhatIsGIINFormProvider()()
 
   ".value" - {
 
-    val fieldName = "value"
+    val fieldName      = "value"
+    val requiredKey    = "whatIsGIIN.error.required"
+    val lengthKey      = "whatIsGIIN.error.length"
+    val invalidKey     = "whatIsGIIN.error.invalid"
+    val invalidCharKey = "whatIsGIIN.error.char"
```

Comment:
There's no test for `whatIsGIIN.error.format`?

Reply:
Added